ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDCase.java

```diff
@@ -63,4 +68,16 @@
     private String features;
     private CCDYesNoOption migratedFromClaimStore;
     private String caseName;
+    private LocalDate docUploadDeadline;
+    private CCDDirectionPartyType docUploadForParty;
+    private LocalDate eyewitnessUploadDeadline;
+    private CCDDirectionPartyType eyewitnessUploadForParty;
+    private List<CCDOrderDirectionType> directionList;
+    private List<CCDOrderDirection> otherDirectionList;
+    private CCDYesNoOption hearingIsRequired;
+    private String newRequestedCourt;
```

Comment:
Are we having multiple requests for the court? as this name suggests there is one existing requests as well